### PR TITLE
replaces Moment with Luxon in controllers and components in ilios-common - part 1

### PR DESCRIPTION
**Pull request by stopfstedt**
refs https://github.com/ilios/ilios/issues/5193

### CODE REVIEW

File: packages/ilios-common/addon/components/sessions-grid-loading.hbs [hunk 1]
```diff
@@ -24,7 +24,7 @@
         {{random 1 99}}
       </span>
       <span class="session-grid-first-offering">
-        {{moment "2005-06-24" "L LT"}}
+        {{format-date (now)}}
```
Comment (jrjohnson) [inline packages/ilios-common/addon/components/sessions-grid-loading.hbs:27]: I can't figure out where the `now` helper is coming from. It works, I'd just like to know why...

**Review by jrjohnson — COMMENTED** [review]
One question... otherwise yeehaw!

File: packages/ilios-common/addon/components/sessions-grid-loading.hbs (hunk 1 above)
Comment (stopfstedt) [reply packages/ilios-common/addon/components/sessions-grid-loading.hbs:27]: it's part of `ember-intl`, see 

https://www.npmjs.com/package/ember-intl/v/4.2.0#format-date

Comment (stopfstedt) [reply packages/ilios-common/addon/components/sessions-grid-loading.hbs:27]: ugh, nevermind. this was supposedly removed in v5 of `ember-intl`. we're currently on v6.

https://ember-intl.github.io/ember-intl/docs/legacy/migration-4-0-to-5-0#intl-relativetimeformat

but it's evidently still working. looking into it...

Comment (stopfstedt) [reply packages/ilios-common/addon/components/sessions-grid-loading.hbs]: handled in the most inelegant way possible :white_check_mark:

**Review by jrjohnson — APPROVED** [review]

**Pull request merged**